tools/rna_tools/rna_commender/.shed.yml

```diff
@@ -0,0 +1,9 @@
+name: rnacommender
+owner: rnateam
+description: INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.
```

Comment:
Done

On Mon, May 30, 2016 at 9:38 PM, Björn Grüning <notifications@github.com>
wrote:

> In tools/rna_tools/rna_commender/.shed.yml
> <https://github.com/bgruening/galaxytools/pull/349#discussion_r65101688>:
>
> > @@ -0,0 +1,9 @@
> > +name: rnacommender
> > +owner: rnateam
> > +description: INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.
>
> @gianlucacorrado <https://github.com/gianlucacorrado> can you please
> adopt this file and insert some useful information here? You can do this
> via the web-interface.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bgruening/galaxytools/pull/349/files/20ad002b457032f67d64cd70ae00d6e0d401b645#r65101688>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQh4OxMrBgz_9fXn7R5HFfjiODVUbi8tks5qGzypgaJpZM4Ip_Ra>
> .
>



-- 
--------------------------------------
Gianluca Corrado
corrag@gmail.com